src/Physics/physicsImpostor.ts

```diff
@@ -39,6 +40,23 @@ export interface PhysicsImpostorParameters {
      * Specifies if bi-directional transformations should be disabled
      */
     disableBidirectionalTransformation?: boolean;
+    /**
+     * The pressure inside the physics imposter, softbody only

```

Comment:
This is where we should mention that only ammo supports it

Reply:
Will do

Reply:
Done